src/pages/ReimbursementAccount/ContinueBankAccountSetup.js

```diff
@@ -44,55 +43,67 @@ const propTypes = {
-        <FullPageNotFoundView shouldShow={_.isEmpty(props.policy)}>
-            <HeaderWithCloseButton
-                title={props.translate('workspace.common.bankAccount')}
-                subtitle={props.policyName}
-                onCloseButtonPress={Navigation.dismissModal}
-                onBackButtonPress={Navigation.goBack}
-                shouldShowGetAssistanceButton
-                guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
-                shouldShowBackButton
-            />
-            <ScrollView style={styles.flex1}>
-                <Section
-                    title={props.translate('workspace.bankAccount.almostDone')}
-                    icon={Illustrations.BankArrow}
-                >
-                    <Text>
-                        {props.translate('workspace.bankAccount.youreAlmostDone')}
-                    </Text>
-                    <Button
-                        text={props.translate('workspace.bankAccount.continueWithSetup')}
-                        onPress={props.continue}
-                        icon={Expensicons.Bank}
-                        style={[styles.mv4]}
-                        iconStyles={[styles.buttonCTAIcon]}
-                        shouldShowRightIcon
-                        large
-                        success
-                    />
-                    <MenuItem
-                        title={props.translate('workspace.bankAccount.startOver')}
-                        icon={Expensicons.RotateLeft}
-                        onPress={() => BankAccounts.requestResetFreePlanBankAccount()}
-                        shouldShowRightIcon
-                        wrapperStyle={[styles.cardMenuItem]}
-                    />
-                </Section>
-            </ScrollView>
-        </FullPageNotFoundView>
+const ContinueBankAccountSetup = (props) => {
+    const errors = lodashGet(props.reimbursementAccount, 'errors', {});
+    const pendingAction = lodashGet(props.reimbursementAccount, 'pendingAction', null);
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <FullPageNotFoundView shouldShow={_.isEmpty(props.policy)}>
+                <HeaderWithCloseButton
+                    title={props.translate('workspace.common.bankAccount')}
+                    subtitle={props.policyName}
+                    onCloseButtonPress={Navigation.dismissModal}
+                    onBackButtonPress={Navigation.goBack}
+                    shouldShowGetAssistanceButton
+                    guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
+                    shouldShowBackButton
+                />
+                <ScrollView style={styles.flex1}>
+                    <Section
+                        title={props.translate('workspace.bankAccount.almostDone')}
+                        icon={Illustrations.BankArrow}
+                    >
+                        <OfflineWithFeedback
```

Comment:
Hi all! Just dropping a note that this PR caused issue - https://github.com/Expensify/App/issues/18517

Wrapping with `OfflineWithFeedback` crosses out all text. In bank account page, it doesn't makes sense to have the strikethrough because the component is static, that is not showing a user input (chat message, workspace etc).

<img width="352" alt="image" src="https://github.com/Expensify/App/assets/29673073/d0c4f6fa-0d7d-4a0c-972a-adeec3b647a9">

Reply:
Nice catch, but with removing the pending action we may lose some other intended strikethough styles?

https://github.com/Expensify/App/pull/15394/files#r1119420530